core/modules/flowgen.cc

```diff
@@ -206,6 +209,18 @@ pb_error_t FlowGen::ProcessArguments(const bess::pb::FlowGenArg &arg) {
     quick_rampup_ = 1;
   }
 
+  ip_src_range_ = arg.ip_src_range();
+  ip_dst_range_ = arg.ip_dst_range();
+  port_src_range_ = (uint16_t)arg.port_src_range();
```

Comment:
Oh that's unfortunate... Then perhaps we can do range check? One may innocently specify 65536 to intend "all ports", but it will result in `port_src_range == 0` due to narrow casting.